src/network/wait-online/manager.c

```diff
@@ -175,9 +172,7 @@ bool manager_configured(Manager *m) {
                         continue;
                 }
 
-                r = manager_link_is_online(m, l,
-                                           (LinkOperationalStateRange) { _LINK_OPERSTATE_INVALID,
-                                                                         _LINK_OPERSTATE_INVALID });
+                r = manager_link_is_online(m, l, NULL);
```

Comment:
maybe `/* state_range= */` before last arg (and then actually renaming the param like that